### ISSUE TITLE
How to choose the best model in the saved models?

### ISSUE THREAD

**Issue by sihuo521** [toandaominh1997/EfficientDet.Pytorch#97]
Thank you very much for sharing.
 Does the data have a validation set? I did not see it.
